src/documentation/install.page

```diff
@@ -12,46 +12,64 @@ lang: en
 
 ## Installing uniqush-push
 
-uniqush-push now is the only component in Uniqush.
+`uniqush-push` is now the only component in Uniqush.
 
-It is higly recommanded to install from **binary** package on **64-bit** system.
+It is highly recommanded to install from **binary** package on **64-bit** system.
 
 ### Install Redis
 
-The only dependency of uniqush-push is [Redis], which is a key-value database.
+The only dependency of `uniqush-push` is [Redis], which is a key-value database.
+
+Redis should be installed on a server which has low latency to the server
+`uniqush-push` is installed on (e.g. the same datacenter). One solution is to
+install both programs on the same server.
 
 - On Debian-based systems (including [Debian], [Ubuntu], [Mint], etc.), run:
-    sudo apt-get install redis-server
-- On yum-based systems (including [Redhat Enterprise Linux], [Fedora], [CentOS], etc.), run:
-    sudo yum install redis-server
+
+    `sudo apt-get install redis-server`
+
+- On yum-based systems (including [Red Hat Enterprise Linux], [Fedora], [CentOS], etc.), run:
+
+    `sudo yum install redis-server`
+
 - For other systems, please visit the [Redis] website for more information.
 
 ### Install uniqush-push
 
-- Download the latest uniqush-push binary package appropriate for your system from the [download page] or you can build it from source. Go 1.6+ is recommended.
+- Download the latest `uniqush-push` binary package appropriate for your system from the [download page] or you can build it from source. Go 1.6+ is recommended.
 - If you are using a Debian-based system, you can install the DEB:
-    > sudo dpkg -i uniqush-push\_VERSION\_ARCH.deb
-- If you are using yum/rpm-based system, you can install RPM:
-    > sudo rpm -ivh uniqush-push-VERSION-ARCH.rpm
+
+    `> sudo dpkg -i uniqush-push\_VERSION\_ARCH.deb`
```

Comment:
The `>` was intended for a markdown block quote. Don't include those, just write "`sudo"